### ISSUE TITLE
Permissions update response for add-ons via group policy

### ISSUE THREAD

**Issue by andygock** [mozilla/policy-templates#592]
When an add-on is updated, it may cause a popup to appear asking the end user whether to accept the changed permissions requested by the add-on.

Is it possible to set via group policy rule, as to whether to default accept/reject/ask etc, so that end users don't have to see the interactive popup?

Thanks.

**Comment by mkaply** [comment]
If I did this, it would only apply to extensions that were explicitly installed by policy or allowed by policy. Does that cover your usecase?

**Comment by andygock** [comment]
@mkaply Yes that would be perfect. It would only need to apply those addons installed by a separate policy rule. Since the addons were installed transparently to the user, it makes sense that updates to it could be made transparent / non interactive as well.

If it can work on both latest release and ESR, would be great. Thanks.

**Comment by mkaply** [comment]
do you happen to have an example of an addon where you saw this? I'm having trouble finding one to test.

**Comment by andygock** [comment]
@mkaply uBlock Origin was one addon that asked for new permissions on a recent update, some of our company users reported the window popping up.

![2020_05_05_09_15_16_Add_ons_Manager](https://user-images.githubusercontent.com/555418/81453859-fb614300-91cd-11ea-97bd-84a59f0a2e58.png)

**Comment by mkaply** [comment]
I opened https://bugzilla.mozilla.org/show_bug.cgi?id=1641093 for this.

**Comment by mkaply** [comment]
This was fixed in Firefox. It will be in Firefox 138 and the next ESR.

See:

https://bugzilla.mozilla.org/show_bug.cgi?id=1907844
